core/src/mender-client.c

```diff
@@ -969,10 +969,10 @@ mender_client_update_work_function(void) {
                     goto END;
                 }
 
-                mender_log_info("Downloading deployment artifact with id '%s', artifact name '%s' and uri '%s'",
-                                deployment->id,
-                                deployment->artifact_name,
-                                deployment->uri);
+                mender_log_debug("Downloading deployment artifact with id '%s', artifact name '%s' and uri '%s'",
+                                 deployment->id,
+                                 deployment->artifact_name,
+                                 deployment->uri);
```

Comment:
I think this one should be info. Our PO has claimed several times that to have progress updates is a highly demanded feature.

Reply:
But I guess the above is a nice first message. :+1:

Reply:
Sorry, I got confused!

I didn't implicitly ask for restoring this one, but it makes sense as it gives context to the other ones (the progress)